src/Storages/System/attachSystemTables.cpp

```diff
@@ -64,6 +64,10 @@
 #include <Storages/System/StorageSystemStackTrace.h>
 #endif
 
+#if __has_include("StorageSystemTimeZones.generated.cpp")
```

Comment:
But it's not include...

Reply:
right, I missed adding that cmake target to include this file after build. thanks for fixing.